packages/npm2gpr/src/index.js

```diff
@@ -0,0 +1,62 @@
+// additional library used to fetch data for a given url
+const fetch = require("node-fetch");
+
+/**
+ * Get the correct url format 
+ */
+function getSearchUrl(org, from = 0) {
+  return `https://registry.npmjs.org/-/v1/search?text=@${org}&from=${from}`;
+}
+
+/**
+ * getting the response and convert into json format and return results
+ * @param {string} org npm organization name 
+ * @param {int} from npm page count
+ */
+async function getSearchPage(org, from = 0) {
+  // return response of given org name and starting object count
```

Comment:
We don't need line-by-line comments. Too many comments can make it difficult to read the code.